.swiftlint.yml

```diff
@@ -124,12 +124,6 @@ comma_inheritance: warning
 # if, for, guard, switch, while, and catch statements shouldn’t unnecessarily wrap their conditionals or arguments in parentheses.
 control_statement: warning
 
-# Limits the complexity of function bodies.
-cyclomatic_complexity:
-  warning: 12
-  error: 20
```

Comment:
What if we just increase these limits to some higher values? I think it's still good to keep functions simple.

Reply:
Maybe bump each by 8?